app/components/DR/ActionCards/ActionCards.js

```diff
@@ -29,7 +29,9 @@ export function Feels({ navigation }) {
           </Text>
           <View style={{ justifyContent: 'center' }}>
             <Button
-              onPress={() => navigation.navigate('ReportScreen')}
+              onPress={() =>
+                navigation.navigate(languages.t('navigation.report'))
```

Comment:
The value that receives the navigate is the ScreenName
this will cause an error

Reply:
Fixed. Now they have a tabBarLabel property and a constant name to ref the screen.